crts/microsoft/desktop/desktop-14.29.30037.json

```diff
@@ -11,7 +11,8 @@
     "target:x86": {
       "requires": {
         "crts/microsoft/desktop/x86": "14.29.30037",
-        "crts/microsoft/onecore/x86": "14.29.30037"
+        "crts/microsoft/onecore/x86": "14.29.30037",
+        "crts/microsoft/store/x86": "14.29.30037"
```

Comment:
Desktop needs store? 😭

Reply:
Yes. :-( 
The solution needs to link with comsuppw[d].lib, which is shipped with VS in the Store .vsix (but which are installed into the same place under %VCINSTALLDIR% as desktop libs).